mysql/mysql.go

```diff
@@ -41,20 +41,30 @@ import (
 )
 
 const (
-	name       = "mysql"
-	version    = 7
-	pluginType = plugin.PublisherPluginType
+	name            = "mysql"
+	version         = 8
+	pluginType      = plugin.PublisherPluginType
+	usernameDefault = "root"
+	passwordDefault = "root"
+	hostnameDefault = "localhost"
+	tcpPortDefault  = "3306"
+
+	databaseDefault = "SNAP_TEST"
+	tableDefault    = "info"
+	tableColumns    = "(timestamp VARCHAR(200), source_column VARCHAR(200), key_column VARCHAR(200), value_column VARCHAR(200))"
```

Comment:
I think that it is good idea but in my opinion it should be done in separated pull request. This PR is just to fix issue #2. What do you think about opening the issue for changing data type of metrics?